src/blog/migrations/0006_authorindexpage_blogpage_author_authorpage.py

```diff
@@ -10,7 +10,6 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('blog', '0005_alter_blogpage_date'),
-        ('wagtailcore', '0093_uploadedfile'),
```

Comment:
Why did you remove this? It's generally a bad sign if you change migrations afterwards 🤔

Reply:
It wouldn't migrate. It gave an error that these two dependencies did not exist 😅

Reply:
Let me see if I can re-produce it before we merge it. I'll get back on it